packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -0,0 +1,75 @@
+import { SelectableValue } from '@grafana/data';
+
+export enum AxisPlacement {
+  Auto = 'auto', // First axis on the left, the rest on the right
+  Top = 'top',
+  Right = 'right',
+  Bottom = 'bottom',
+  Left = 'left',
+  Hide = 'hide',
+}
+
+export function getUPlotSideFromAxis(axis: AxisPlacement) {
+  switch (axis) {
+    case AxisPlacement.Top:
+      return 0;
```

Comment:
Perhaps there is some better way?
https://github.com/leeoniya/uPlot/blob/master/dist/uPlot.d.ts#L461

Reply:
i haz to expoze moar tipes!